web/css/mobile_menu.css

```diff
@@ -0,0 +1,123 @@
+#menupanel {
+    background-color: white ;
+    display : none ;
+    position: fixed;
+    display: none;
+    width: 80%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 3 ;
+    
+}
+
+#overlay {
+    position: fixed;
+    display: none;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0,0,0,0.6);
+    z-index: 2;
+    cursor: pointer;
+}
+
+.profile {
+    background:  url('../images/backgrounds/wallpaper.jpg') no-repeat;
```

Comment:
Beware that line, I think the old wallpaper is too heavy to be loaded on mobile devices. It should be removed